storybook/README.md

```diff
@@ -0,0 +1,322 @@
+
+The `sbs` (“Storybook story”) will scaffold the entire code in the [section above](#scaffold-tour). If you have respected the naming conventions, it will also property import your component and the properties with the correct names, saving you a bunch of time.
+
+The `sbt` (“Storybook story template”) will create a new story by binding your default template and prompting you to provide specific arguments.
+
+## FAQ
+
+### What should constitute a story?
+
+Stories should show the component in its different shapes.
+
+The most obvious way is when a component has a few variations (such as a button with a `primary` attribute being larger and bolder, for example).
+
+However, a common misconception is that, since full controls are enabled and allow the user to explore the component by playing around with its properties interactively, there is no need to provide stories which are just the component with different default properties.
+
+While this is true for many cases (often you don't need to create a different story by replacing the `label` of a component, if that's just text; though it might be argued that it could be interesting to have a story showing how a very long text behaves, especially in [combined stories](#can-i-create-stories-with-mixed-components)), it is very useful to create stories for components in their loading or error states.
+
+### Can I create stories with mixed components?
+
+Yes, and it'd be awesome to see how our components interact with each other, especially in the context of blocks. At the time of writing, we have no such stories, however, here is a link to the official docs if you want to give a go at implementing this: https://storybook.js.org/docs/react/workflows/stories-for-multiple-components
+
+## Tips
+
+### One file per component
+
+While stories for closely related components could technically be living within the same file, we advise that you keep one file per component, as this works best with the automatic organization of the components within storybook and using [`default` exports in the Story](https://storybook.js.org/docs/react/api/csf#default-export).
+
+An exception to this, might be when there is only one mainly used component and the other defined ones are just used internally.
+
+E.g. compare `Chip/RemovableChip` with `ProductPrice/Sale/Range`. In the latter case, `Sale` and `Range` components are actually rendered depending on the props passed to the `ProductPrice` component, so they might as well be stories of their parent component.
+
+### Custom controls
+
+Sometimes the inferred text input is not good enough, and you need something more complex. If those controls happen to be shared among many components, please write them in `storybook/custom-controls`.
+
+An example there is the `currency` control. Since many of our components expect `Currency` objects in their props, we can make sure we give a few example Storybook users can play with, without having to manually create the object themselves.
```

Comment:
```suggestion
An example there is the `currency` control. Since many of our components expect `Currency` objects in their props, we can make sure we give a few examples Storybook users can play with, without having to manually create the object themselves.
```